appintro/src/main/java/com/github/paolorotolo/appintro/AppIntroBase.java

```diff
@@ -107,12 +107,6 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         checkButton(skipButton, "skip");
         checkButton(backButton, "back");
 
-        FrameLayout frameLayout = findViewById(R.id.bottomContainer);
-        if (frameLayout != null && isRtl()) {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
-                frameLayout.setLayoutDirection(View.LAYOUT_DIRECTION_RTL);
```

Comment:
So removing this is breaking `rtl` on the dot indicator.
Please make sure to test your UI in the RTL condition (there is a toggle in the Developer options), and it should work properly :)

Reply:
Done